x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section_results.tsx

```diff
@@ -217,6 +229,18 @@ export const ExpandableSectionResults: FC<ExpandableSectionResultsProps> = ({
     [indexData?.visibleColumns, discoverLocator, dataViewId, resultsField, tableItems, data]
   );
 
+  const openCustomUrl = (item: DataGridItem, customUrl: KibanaUrlConfig) => {
+    // eslint-disable-next-line no-console
+    console.log('Data Frame Analytics Results Table - open customUrl for record:', item);
+    // Replace any tokens in the configured url_value with values from the source record and open link in a new tab/window.
+    const urlPath = replaceTokensInDFAUrlValue(
+      customUrl,
+      item as any,
```

Comment:
can we get rid of any?

Reply:
Removed in 642ce6dbe70bb719b6416e5a61184a6b966b71d7